security/models.py

```diff
@@ -50,7 +50,7 @@ def display_json(value, indent=4):
 
 def hide_sensitive_data_body(content):
     for pattern in settings.HIDE_SENSITIVE_DATA_PATTERNS.get('BODY', ()):
-        content = regex_sub_groups_global(pattern, settings.SENSITIVE_DATA_REPLACEMENT, content)
+        content = regex_sub_groups_global(pattern, settings.SENSITIVE_DATA_REPLACEMENT, content, re.IGNORECASE)
```

Comment:
víš že se dá v těch regexech rovnou nastavit aby byly case insensitive re.search(r'(?i)test', 'TeSt').group()  proto jsem to také nedělal. Umožňujeme tu tím aby si mohl vývojář vybrat

Reply:
I think in this case, it is better to accidentally hide more than hide less. Can you think of any case where  case insensivity would make problems? The truth is, the current state is more flexible.

Reply:
měli jsme nějaké issue, kde to bylo potřeba. Proto tohle nechci